### DISCUSSION TITLE
Can you remove tabl-caps fully?

### DISCUSSION THREAD

**Discussion by IceManTBM3** [quarto-dev/quarto-cli#7663] [Q&A] [answered]
I am wondering if there is a way to cull the tabl-cap title in tabulated tables. If I pull the R code for tbl-cap form the code below, I get a generic "Table 99" sub cap (see pic Cap2), can this be sub title be totally removed? Have tried a few things from the Quarto doc and AI bots but nothing seems to work to that end, thx in advance.

>>>>images of the results Cap1 the code above and Cap2 the results without the "#| tbl-cap: "Sovereign Carbon Field Technical Details"" line:
![cap2](https://github.com/quarto-dev/quarto-cli/assets/129208946/3fee4150-3664-4638-9818-fadaf64286ef)
![cap1](https://github.com/quarto-dev/quarto-cli/assets/129208946/142a954e-b1f7-46fa-8271-5dfd1bf33ec2)

here's the R code, pretty generic.
```{r}
#| label: tbl-sov-carbon-technical
#| tbl-cap: "Sovereign Carbon Field Technical Details"
#| echo: false

**Comment by mcanouil** [comment]
I am sorry but I don't understand at all what you are saying, what you want to get and from what.
Also please fill in properly the discussion with a title and proper code/content formatting.
Next time, please take 2 minutes to check that what you post is legible. Thank you for your understanding.

Note that sharing a small self-contained "working" (reproducible) example to work with increase ten-fold the likelihood of you getting an answer that is not this one, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by IceManTBM3** [reply]
Look at the 2 attachments.
- Cap1 has a the table sub-caption "Table 26 Sovereign Carbon Field Technical Details"
- I pull the R code "#| tbl-cap: "Sovereign Carbon Field Technical Details"" and in Cap2, I expect to have NO sub caption but I have "Table 26" ... is there a way to get rid of that?

**Reply by mcanouil** [reply]
Please format the title and the content of your post and share a complete reproducible example as requested.

Note that this is not a personal request and definitely not to annoy you.
Currently, the content and the title is not legible thus this forbid other users to read, help or use anything from it.

**Reply by IceManTBM3** [reply]
hopefully that helps?

**Reply by cscheid** [reply]
Sorry, it really doesn't. We can't take that and run `quarto render` it ourselves. We need to see the *entirety* of your document to be able to help.

**Reply by mcanouil** [reply]
@IceManTBM3 
<img width="1329" alt="Screenshot 2023-11-21 at 18 47 06" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/06404e8d-322d-4cdc-a594-9b4dac069a83">

And regarding what is a reproducible example, I will refer you to my first reply: <https://github.com/quarto-dev/quarto-cli/discussions/7663#discussioncomment-7632990>.

**Reply by IceManTBM3** [reply]
This really has nothing to do with a code sample really can YOU make table with multiple tabs that does not have sub-tab labels?

Is that possible?

**Reply by mcanouil** [reply]
@IceManTBM3 Please could you just update/edit your post and title for it to be legible to all users!

Use the edit buttons and make your post legible.
<img src="https://github.com/quarto-dev/quarto-cli/assets/8896044/9bbdeb6b-5158-48a2-ba2a-2802746d973b" width="450px"></img>

If for some reason, you really cannot edit yourself, we'll see.
I am afraid, until you comply with this basic accessibility request, you won't get an answer.

**Reply by IceManTBM3** [reply]
what is unclear to you? 

this is the Q: can YOU make table with multiple tabs that does not have sub-tab labels?

is that not clear?

also too, if you want more, please be specific

**Reply by cscheid** [reply]
@IceManTBM3 Mickael helps literally dozens of people every day in this forum. I want to ask you here to try and be better, and actually answer his question. There is a reason we ask you to provide these details, and that is to save everyone time. Please do address his request.

This is relatively rude behavior on your part, and that repeated incidents of this run afoul of our code of conduct.

**Reply by IceManTBM3** [reply]
how has that run afoul or is rude? I simply restated the question to be more discreet and to the point to add clarity. I then said if that's not clear, to ask specifics. It is frustrating when someone says you are not clear, and then you restate the question, and they say it's "illegible" and that is my Q,  WHAT is illegible? Also I would say invoking the term "illegible" bears a negative context or is rude in most uses. I've gotten plenty of good help here and hope that continues. 

again, the Q  is: can YOU make a table with multiple tabs that does not have sub-tab labels?

Can you suppress them?

To answer yor question I did edit the Question as requested. Is there something else that is specially needed? If so please restate bc I am not clear.

thanks and have a nice day

**Reply by mcanouil** [reply]
> how has that run afoul or is rude?

The fact that you deliberately ignored any request we've made!
We've got your question, but apparently you don't want at all to address the accessibility request that was made several times.
I added screenshots to show you what to do but you still don't want to comply with this very basic thing.

> [!CAUTION]
> **So, one last time: please edit the title of your post and edit the content to actually make your screenshots show!**

In case you don't know what post I am talking about, I am referring to the very first one: https://github.com/quarto-dev/quarto-cli/discussions/7663#discussion-5876913

Again here is the screenshot showing exactly what's wrong and what we are asking you to do!
<img width="800" alt="Screenshot 2023-11-21 at 18 47 06" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/06404e8d-322d-4cdc-a594-9b4dac069a83">

Use the edit buttons and make your post legible.
<img src="https://github.com/quarto-dev/quarto-cli/assets/8896044/9bbdeb6b-5158-48a2-ba2a-2802746d973b" width="450px"></img>

**Reply by IceManTBM3** [reply]
ah got you I edited the wrong part, thx.... did this form change recently?

I also move the images down...

but the distinct Q is, can these sub caps be removed?  It's not imperative, but it could make for a less cluttered display

**Reply by mcanouil** [reply]
This is a nearly native thing.
Nothing changed!

Thanks for the title that now search engine and people can read and use.
Please also edit your post to fix the formatting to actually show your attachments. (I am not going to request for alternative text but know that without people with disabilities won't be able to "see" your images)

Note that the syntax to format code is also shown in my very first reply: https://github.com/quarto-dev/quarto-cli/discussions/7663#discussioncomment-7632990

This is regular markdown as you use in Quarto.
So close the code block and the images will show.

**Reply by IceManTBM3** [reply]
ahhh I see, think I got it.

**Reply by mcanouil** [reply]
Please be aware that because you did not read our comments you cost us time which could have been used to help you and others.
Taking two minutes to verify your post after writing it could have save a lot of time to everyone.

**Reply by IceManTBM3** [reply]
^^ friendly note, using language like "deliberately ignored" and "rude" without a personal attack or foul language is assigning intent and confrontational, maybe rethink that as folks come at these things differently, I'm not a techie so Git and coding is all something I use as part of my job but not my area of expertise. So I guess, don't assume, miscommunication happens.

Cheers and glad we got through it...

**Reply by mcanouil** [reply]
> ^^ friendly note, using language like "deliberately ignored" and "rude" without a personal attack or foul language is assigning intent and confrontational, maybe rethink that as folks come at these things differently

"friendly note": Do you realise how many comments, including screenshots were made?

> I'm not a techie so Git and coding is all something I use as part of my job but not my area of expertise.

"friendly note": Reading comments and screenshots is not "techie".
Proof reading what you write is not "techie".

Everything was in my first reply including the exact syntax to share code, copy/pasting it to add your "code" in it is not that "techie".

> Also please fill in properly the discussion with a title and proper code/content formatting.

This was plain english, nothing "techie".


> [!TIP]
> In the future, please do:
> - double check what you are writing/posting.
> - provide a small reproducible example as shown.
>
> Thank you for your understanding and for using Quarto!

---

## Answer to the question (also posted as a separate thread since all this will be hidden in a near future as "off topic")

All this being said:
1. there is no "tab subcap" in your screenshots or in your code.
2. If you do not want your table to have `Table <number>: <caption>`, do not make it a cross-reference object (see <https://quarto.org/docs/authoring/cross-references.html#tables>), *i.e.*, using `#| label: tbl-` means "I want to have `Table <number>: <caption>`.

Now a reproducible example you can actually use and that we've asked from you that I had to make myself (nothing "techie" here, especially if you intend to use Quarto):

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr>
<td>

````qmd
---
title: "Issue"
format:
  html:
    embed-resources: true
---

```{r}
#| label: sov-carbon-technical
#| tbl-cap: "Sovereign Carbon Field Technical Details"
#| echo: false

knitr::kable(head(mtcars))
```
````

</td>
<td>
<img width="574" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/798ee786-7434-4a12-8a4c-38df6c603ff9">
</td>
</tr>
</table>

**Reply by IceManTBM3** [reply]
Gentlemen, am I missing something or is this area not supposed to be communal and collaborative? This insistence on “you did this and that” and rehashing the matter does not advance the conversation constructively or bring answers, and now with the antagonistic tagging of “friendly note” and “techie” borders on uncivil. Is this how Posit does business with users and customers?

**Comment by mcanouil** [comment, marked as answer]
## Answer to the question

All this being said:
1. there is no "tab subcap" in your screenshots or in your code.
2. If you do not want your table to have `Table <number>: <caption>`, do not make it a cross-reference object (see <https://quarto.org/docs/authoring/cross-references.html#tables>), *i.e.*, using `#| label: tbl-` means "I want to have `Table <number>: <caption>`.

Now a reproducible example you can actually use and that we've asked from you that I had to make myself (nothing "techie" here, especially if you intend to use Quarto):

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr>
<td>

````qmd
---
title: "Issue"
format:
  html:
    embed-resources: true
---

```{r}
#| label: sov-carbon-technical
#| tbl-cap: "Sovereign Carbon Field Technical Details"
#| echo: false

knitr::kable(head(mtcars))
```
````

</td>
<td>
<img width="574" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/798ee786-7434-4a12-8a4c-38df6c603ff9">
</td>
</tr>
</table>
